bluepyemodel/access_point/__init__.py

```diff
@@ -36,7 +36,7 @@ def get_access_point(access_point, emodel, **kwargs):
 
     For local:
         emodel_dir (str): path of the directory containing the parameters,
-            features and parameters config files.
```

Comment:
Does the doc understand that this line is not a new attribute, but the end of the description of emodel_dir?

Reply:
Good catch, I had to use hyphen to correctly indent the optional parameters, should work now